src/ic-websocket-gateway/src/client_session_handler.rs

```diff
@@ -186,26 +186,26 @@ impl ClientSessionHandler {
                     client_session_span.in_scope(|| {
                         debug!("Client session opened");
 
+                        let client_key = self.get_client_key(&client_session);
+
                         gauge!("clients_connected").increment(1.0);
 
-                        let client_key = self.get_client_key(&client_session);
-                        let key = format!("client_{}", client_key.clone().to_string());
-                        clients_session_time.insert(key, Instant::now());
+                        clients_session_time.insert(client_key.clone().to_string(), Instant::now());
```

Comment:
I don't think we need to serialize the `client_key` to a `String` here.

Reply:
For clarity, on line 112 you could explicitly declare the HashMap type and make it look something like:
```rust
let mut clients_session_time: HashMap<ClientKey, Instant> = HashMap::new();
```